### PR TITLE
Statistics Added!

### PR DESCRIPTION
**Pull request by tailaiwang**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,600 @@
...
+  <section id="about" class="container mt-5 mb-5 pt-5 pb-5">
+    <div class="row align-items-center">
+      <div class="col-md">
+        <img src="static/code.svg" class="image" />
+      </div>
+
+      <div class="col-md mx-1 my-5">
+        <h2 class="text-center mb-3"># What is The GoldenHack?</h2>
+        <p class="text-center">
+          <b>The GoldenHack</b> is a business hackathon held at Laurier on<br />
```
Comment (vipj) [inline index.html]: I think it should be "... held at Laurier in October 2020."

Comment (dunja-tomic) [reply index.html]: @tailaiwang 👀

Comment (tailaiwang) [reply index.html]: Fixed it in the latest commit!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,600 @@
...
+
+  </section>
+  <hr>
+
+  <section id="statistics">
+    <h2 class="text-center mb-3"># Last Year we Had...</h2>
+    <div class="container align-items-center">
+      <div class = "row align-items-center">
+        <div class = "col align-items-center" style="text-align: center;">
+          <div class="circle" style="color: white; background: #053E5E;">
```
Comment (dunja-tomic) [inline index.html:319]: technically this styling should be in a css class of its own but eh it's fine we'll have a new version of the website up soon enough

Comment (tailaiwang) [reply index.html:319]: Ohh I gotcha now. I'll keep that in mind as we go on.

**Pull request merged**